lib/ui/pages/account_dashboard.dart

```diff
@@ -0,0 +1,351 @@
+import 'package:pispapp/controllers/ephemeral/account_dashboard_controller.dart';
+import 'package:pispapp/models/account.dart';
+import 'package:pispapp/models/transaction.dart';
+import 'package:pispapp/ui/theme/light_theme.dart';
+import 'package:pispapp/ui/widgets/icon.dart';
+import 'package:pispapp/ui/widgets/title_text.dart';
+import 'package:pispapp/utils/utils.dart';
+
+class AccountDashboard extends StatelessWidget {
+  Widget _getAccountTile(Account acc) {
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(20, 0, 20, 0),
+      child: ListTile(
+        onTap: () {
+          Get.find<AccountDashboardController>().onAccountTileTap(acc);
+        },
+        leading: const CircleAvatar(),
+        contentPadding: const EdgeInsets.symmetric(),
+        title: TitleText(
+          text: acc.alias,
+          fontSize: 14,
+        ),
+        subtitle: Text(Utils.getSecretAccountNumber(acc)),
+      ),
+    );
+  }
+
+  void _showAccountChoosingBottomSheet() {
+    Get.bottomSheet<void>(
+      Container(
+        height: 350.0,
+        color: const Color(0xFF737373),
+        child: Container(
+          decoration: const BoxDecoration(
+            color: Colors.white,
+            borderRadius: BorderRadius.only(
+              topLeft: Radius.circular(
+                20.0,
+              ),
+              topRight: Radius.circular(
+                20.0,
+              ),
+            ),
+          ),
+          child: GetX<AccountController>(
+            builder: (value) {
+              return ListView(
+                children: <Widget>[
+                  const Padding(
+                    padding: EdgeInsets.all(
+                      20.0,
+                    ),
+                    child: TitleText(text: 'Accounts'),
+                  ),
+                  const Divider(
+                    height: 20,
+                  ),
+                  for (var acc in value.accounts.value) _getAccountTile(acc)
+                ],
+              );
+            },
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _appBar() {
+    return Row(
+      children: <Widget>[
+        const CircleAvatar(),
+        const SizedBox(width: 15),
+        GetBuilder<AccountDashboardController>(
+          builder: (_) {
+            return TitleText(text: _.selectedAccount.alias);
+          },
+        ),
+        const Expanded(
+          child: SizedBox(),
+        ),
+        GestureDetector(
+          onTap: () {
+            _showAccountChoosingBottomSheet();
+          },
+          child: Icon(
+            Icons.short_text,
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _operationsWidget() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceAround,
+      children: <Widget>[
+        CustomIcon(Icons.delete_sweep, 'Unlink', () {}),
+        CustomIcon(Icons.code, 'Qr Pay', () {}),
+      ],
+    );
+  }
+
+  void _showTransactionDetailBottomSheet(Transaction t) {
```

Comment:
it would be better to create a widget for this

Reply:
Again this is something we use only in this widget. In case we need it at multiple places, it's better to move it out, but I highly doubt it.

Reply:
yeah, but i think it would be better to avoid mixing the widget definition, which is only for the purpose of display, with the page layout which also contain some logic to update the widgets

Reply:
I think that's not a big issue, just to separate the layout + logic with the ui display.
It would be great to follow the SR and SoC principles :)

Reply:
addressed in https://github.com/mojaloop/pisp-demo-app-flutter/pull/12